src/app/analyze/analyze-overview/analyze-overview.component.html

```diff
@@ -1,151 +1,252 @@
+                  <option value="AZ">Pipeline 3</option>
+                  <option value="AR">Pipeline 4</option>
+                  <option value="CA">Pipeline 5</option>
+                  <option value="CO">Pipeline 6</option>
+                  <option value="CT">Pipeline 7</option>
+                  <option value="DE">Pipeline 8</option>
+                </select>
+              </div>
+            </form>
+            <ul class="list-view-pf analytical-report-list">
+              <li class="list-group-item">
+                <div class="list-group-item-text">
+                  <a>Build #102</a>
+                  <span> - You have used</span>
+                  <a>Java JDK 2.0.3</a>
+                  <span>better way to run/deploy you can use</span>
+                  <a>Java JDK 2.0.8</a>
                 </div>
-            </div>
-        </div>
-        <div class="col-md-6">
-            <div class="row">
-                <div class="panel panel-140 panel-default">
-                    <div class="panel-body">
-                        <form class="form-horizontal" (ngSubmit)="saveDescription()" role="form">
-                            <div class="form-group">
-                                <div class="col-sm-12">
-                                    <textarea name="description" id="description" class="form-control" placeholder="Project description. Click to edit..." rows="4"></textarea>
-                                </div>
-                            </div>
-                            <div class="form-group">
-                                <div class="col-sm-12">
-                                    <button type="submit" class="btn btn-default pull-right" value="0">Save</button>
-                                </div>
-                            </div>
-                        </form>
-                    </div>
+              </li>
+              <li class="list-group-item">
+                <div class="list-group-item-text">
+                  <a>Build #102</a>
+                  <span> - We recommend to use you can use</span>
+                  <a>Java JDK 2.0.8</a>
                 </div>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </aside>
+    </div>
+  </section>
+  <section class="row row-cards-pf space-overview-wrapper">
+    <div class="flex-grow-4 " >
+      <aside>
+        <div class="card-pf">
+          <div class="flex-grow-1 flex-container in-column-direction">
+            <h2 class="card-pf-title card-pf-utilization-details">
+              <span>My Work Items</span>
+              <span class="badge">20</span>
+            </h2>
+            <div class="blank-slate-pf">
+              <div class="blank-slate-pf-icon">
+                <span class="pficon pficon pficon-add-circle-o"></span>
+              </div>
+              <h1>Create Work Item</h1>
+              <p>Planner will help you to create different
```

Comment:
this copy needs to be reviewed for context and grammar. @essjaysee, can you take a look?